rest-server/src/gSoapFiles/CAP/soapH.h

```diff
@@ -922,9 +923,9 @@ inline _ns2__alert_info_area_geocode * soap_new__ns2__alert_info_area_geocode(st
 }
 
 inline _ns2__alert_info_area_geocode * soap_new_req__ns2__alert_info_area_geocode(
-	struct soap *soap,
-	const std::string& valueName,
-	const std::string& value)
+		struct soap *soap,
```

Comment:
There's a lot of non-functional changes from here down. Just indentations from what I can tell. Is there a reason for them?

Reply:
done